### PR TITLE
feat: ensure arpa_reporter users are redirected to ARPA tool upon user creation

### PR DESCRIPTION
**Pull request by as1729**
### Ticket #404
https://github.com/usdigitalresponse/usdr-gost/issues/404

### Description
Currently the welcome email for all new users defaults to the ID tool. This PR ensures that when users are being created for the ARPA reporter tool-- then they receive an email redirecting them to the ARPA tool.

### Screenshots / Demo Video
https://www.loom.com/share/35eb95346f1448c38919660de7c9f951

### Testing
TODO: add unit test

### Checklist
- [x] Provided ticket and description
- [x] Your PR title contains the ISSUE ticket number e.g "86: ..."
- [x] Provided screenshots/demo
- [ ] Provided testing information
- [ ] Provided adequate test coverage for all new code
- [ ] Added PR reviewers
- [ ] Ensure at least 1 review before merging

### CODE REVIEW

File: packages/server/src/arpa_reporter/routes/users.js [hunk 1]
```diff
@@ -67,7 +67,13 @@ router.post('/', requireAdminUser, async function (req, res, next) {
       const updatedUser = await createUser(user)
       res.json({ user: updatedUser })
 
-      void sendWelcomeEmail(updatedUser.email, req.headers.origin)
+      let origin = req.headers.origin;
+
+      if (req.headers.referer.indexOf('arpa_reporter')) {
```
Comment (mattbroussard) [inline packages/server/src/arpa_reporter/routes/users.js:72]: nit: this route is ARPA Reporter-specific anyway, so this referer check is not necessary

**Review by mattbroussard — APPROVED** [review]

File: packages/server/src/arpa_reporter/routes/users.js (hunk 1 above)
Comment (mattbroussard) [reply packages/server/src/arpa_reporter/routes/users.js:72]: also, if you are going to keep it, you need to check `!== -1` because `-1` is truthy

Comment (mattbroussard) [reply packages/server/src/arpa_reporter/routes/users.js:72]: also, does this NPE if there is no Referer header? Might need an optional chain

Comment (as1729) [reply packages/server/src/arpa_reporter/routes/users.js:72]: great point, I'll go ahead and remove this conditional check entirely. I was wondering whether the `arpa-annual-reporter` endpoint uses the same route so wanted add some type of `if/else` logic.

Comment (as1729) [reply packages/server/src/arpa_reporter/routes/users.js]: @mattbroussard can you re-review I have resolved the issue by removing the unnecessary conditional.

Comment (mattbroussard) [reply packages/server/src/arpa_reporter/routes/users.js]: >I was wondering whether the arpa-annual-reporter endpoint uses the same route
That tool is only a single page and doesn't have its own user admin page etc. -- it just uses GOST's.

>can you re-review
fwiw, "approved with comments" generally means the comments are minor enough that reviewer trusts you to make the changes and merge without another review round (still feel free to ask for additional review if you substantially changed the PR but this change is pretty minor)

**Pull request merged**